src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,333 @@
+
+### Begin The Revolution
+
+Your associates gather the tools needed to overthrow the station and expose more people to your influence, making your conversion job easier. You sit like a spider in the web and plot.
+
+### Overthrow Centcomm
+
+You steal the Head IDs, break into the bridge and switch off the station’s link to Centcomm, winning the round.
+
+## Core Mechanics
+
+### Revolution Implant
+
+You have a Revolution Implant. This is a really powerful and special implant that hooks into Syndicate mind-control rays being blasted at the station. This implant does nothing outside the Rev round type.
+
+This implant is what makes you a Head Rev. It can be removed via an Implanter, and if someone else is injected with it, they become a Head Rev. This probably will never happen (but would be a really robust way to evade Sec figuring out who the HRs are)
+
+### Communications
+
+Your implant gives you access to the Syndicate radio channel innately. This can be changed to a unique Revolution radio channel, but given Nukies, Traitors and Revs are exclusive to each other, it should be fine to just use the Syndie channel. Head Revs cannot talk to normal Revs on this channel: it's just for planning and collaboration between Head Revs.
+
+### Influence
+
+All non-head and non-security crew have an internal Influence count.
+
+Players gain Influence by:
+
+- Standing near a Head Rev they can see.
+
+- Being near Propaganda they can see.
+
+- Being injected with Mindbreaker Toxin.
+
+Players lose influence by:
+
+- Not being near any source of Influence. This is quite slow, and is reduced to nil if the player has been converted.
+
+- Influence degrades reasonably quickly if placed in the Sec cells and exposed to calming jazz music.
+
+### Conversion
+
+By standing while conscious near people, you increase their Influence to your control. Once they reach a certain number, you can target them with your innate Convert ability. This mentally dominates a loyal crewmember and hands them their shiny Rev antag token. They are not told who converted them; it’s your choice as Head Rev to communicate this information.
+
+Conversion works as follows:
+
+1. The Revolutionary Implant grants the owner the ability to see the Influence level of those around them.
+2. If the Head Revolutionary sees a crew member with a high enough Influence level, they can activate their implant's ability and click on the target.
+3. If successful, the target is silently converted to a Revolutionary.
+4. The ability enters a cooldown.
+
+### Centcomm Uplink Terminal
+
+You win by compromising this terminal.
+
+To trigger this:
+
+1. Get at least one of the Heads ID cards. This must be their special ID cards, not HOP-manufactured ones.
+
+2. Input the cards and click the button to start the unlinking of the uplink.
+
+3. Wait some number of seconds for the uplink to be disabled. This alerts the entire station over comms that the Revs are doing this: I hope you have fortifed the bridge!
```

Comment:
This all seems like a much better way to end the game. Provides crew and revs with a clear goal and space to play around. The use of extra head IDs to speed up/slow down the countdown seems like a perfect tug of war.

Reply:
Only potential issue i CAN see is that it forces the bridge to become a battleground in every instance. Maybe discuss ideas for alternative locations to have this occur, could be any of the head offices so that, for example, revs setup in Medbay could work from CMO's and its then a medical rebellion.

Reply:
At the moment I'm focussing on the Bridge alone so there's a three-step structure to a revolution round: the plotting phase with a small number of Revs, the stealing of Head IDs which is the mid-round goal and then the attack on the Bridge as the end game.

Depending on how this mode works in practice then I could see allowing for the flag to be captured from the various head offices, but I'd worry that'd over-encourage one-department scheming, like a Rev round where the Revs just take over Science and fortify it ridiculously for two hours.

Reply:
At the end of the round there is likely to be a battle anyways and I think it's far better if it's around the bridge rather than just the evac shuttle getting bombed. Using the bridge as a more central objective point helps decrease the focus of antags from the evac shuttle at round end which will reduce shuttle bombings.